2020-04-07 08:30:29,120 - app - INFO - hanka logged in successfully
2020-04-07 08:30:29,121 - app - ERROR - Error
2020-04-07 08:35:09,653 - app - INFO - hanka logged in successfully
2020-04-07 08:35:09,653 - app - ERROR - Error
2020-04-07 08:36:44,639 - app - INFO - hanka logged in successfully
2020-04-07 08:36:44,639 - app - ERROR - Error
2020-04-07 08:43:45,507 - app - INFO - hanka logged in successfully
2020-04-07 08:43:45,508 - app - ERROR - Error
2020-04-07 09:29:37,167 - app - ERROR - Exception on /occur/ [GET]
Traceback (most recent call last):
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Program Files\Python38\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Program Files\Python38\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "app.py", line 140, in occur_get
    return getOccurItems(rpt_id)
  File "app.py", line 417, in getOccurItems
    cur.execute(sql)
pyodbc.ProgrammingError: ('42S22', "[42S22] [Microsoft][ODBC SQL Server Driver][SQL Server]Invalid column name 'CATEGORY_COD'. (207) (SQLExecDirectW)")
2020-04-07 09:30:12,961 - app - ERROR - Exception on /occur/ [GET]
Traceback (most recent call last):
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Program Files\Python38\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Program Files\Python38\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Program Files\Python38\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "app.py", line 140, in occur_get
    return getOccurItems(rpt_id)
  File "app.py", line 417, in getOccurItems
    cur.execute(sql)
pyodbc.ProgrammingError: ('42S22', "[42S22] [Microsoft][ODBC SQL Server Driver][SQL Server]Invalid column name 'CATEGORY_COD'. (207) (SQLExecDirectW)")
2020-04-07 09:32:20,368 - app - INFO - hanka logged in successfully
2020-04-07 14:36:38,858 - app - INFO - hanka logged in successfully
2020-04-07 16:10:57,318 - app - INFO - hanka logged in successfully
2020-04-08 08:16:15,531 - app - INFO - hanka logged in successfully
2020-04-08 08:47:18,440 - app - INFO - admin logged in successfully
2020-04-08 11:50:11,063 - app - INFO - hanka logged in successfully
2020-04-08 11:50:47,300 - app - INFO - admint logged in successfully
2020-04-14 14:41:08,319 - app - INFO - hanka logged in successfully
2020-05-14 18:22:29,311 - app - INFO - hanka logged in successfully
2020-05-19 09:53:52,350 - app - INFO - hanka logged in successfully
2020-05-19 13:32:35,076 - app - INFO - hanka logged in successfully
2020-05-20 08:39:07,690 - app - INFO - hanka logged in successfully
2020-05-20 11:00:28,955 - app - INFO - hanka logged in successfully
2020-05-20 11:01:03,806 - app - INFO - hanka logged in successfully
2020-05-21 07:42:20,928 - app - INFO - hanka logged in successfully
2020-05-21 14:11:18,543 - app - INFO - hanka logged in successfully
2020-05-25 08:13:56,091 - app - INFO - hanka logged in successfully
